src/encode.cpp

```diff
@@ -1063,14 +1068,20 @@ void encodeOp(State &st, mlir::tosa::DepthwiseConv2DOp op, bool) {
                   weight.get({weight2DInd[0], weight2DInd[1], c, m}).first
                 );
 
-  // t is 1xOHxOWx1
-  auto t = input2D.conv(weight2D,
+  // t2D is 1xOHxOWx1
+  auto t2D = input2D.conv(weight2D,
                       strides, dilations, ShapedValue::ConvLayout::NHWC_HWCF);
-  auto tDims = t.getDims();
+  auto t2DDims = t2D.getDims();
+
   // NxOHxOWx(C*M)
-  vector<Expr> outDims = {N, tDims[1], tDims[2], C * M};
+  vector<Expr> tDims = {N, t2DDims[1], t2DDims[2], C * M};
 
-  auto output = addBias2D(elemTy, outDims, t, bias);
+  auto acc = Tensor::mkInitializedLambda(
+            elemTy, move(tDims), move(outInd), 
+            t2D.get({Index(0), outInd[1], outInd[2], Index(0)}).first
+          );
```

Comment:
Why is this added?

Reply:
there is an error in the previous version... forgot to push the commit. The index should be different since t2D is 1xOHxOWx1